rpcs3/rpcs3.vcxproj

```diff
@@ -960,6 +967,16 @@
       <AdditionalInputs Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">$(QTDIR)\bin\moc.exe;%(FullPath);$(QTDIR)\bin\moc.exe;%(FullPath)</AdditionalInputs>
       <AdditionalInputs Condition="'$(Configuration)|$(Platform)'=='Release|x64'">$(QTDIR)\bin\moc.exe;%(FullPath);$(QTDIR)\bin\moc.exe;%(FullPath)</AdditionalInputs>
     </CustomBuild>
+    <CustomBuild Include="rpcs3qt\debugger_add_bp_window.h">
```

Comment:
I don't know if anything changed here since the PR was maybe.
Maybe it's worth double checking if all the params are up to date

Reply:
That part is new so it should be fine, I just copy/pasted entry below and changed the appropriate stuff.